tests/acceptance/17_users/unsafe/20_modify_user_missing_secondary_group.cf

```diff
@@ -0,0 +1,60 @@
+bundle common test_meta
```

Comment:
Instead of defining things in bundle common test_meta, define *meta* type promises inside `bundle agent test`